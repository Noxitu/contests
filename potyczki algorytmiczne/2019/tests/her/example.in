5
2
2 1 4
2 5 2
2
1 4 3
1 5 4
2
1 5 7
1 7 5
2
1 4 1
1 2 5
3
2 6 4
1 2 3
3 4 5