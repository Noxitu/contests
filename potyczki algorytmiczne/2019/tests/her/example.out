TAK
NIE
TAK
NIE
TAK